fabric/build.gradle

```diff
@@ -96,6 +98,10 @@ jar {
 
 shadowJar {
     configurations = [project.configurations.shadowCommon, project.configurations.shadowImplementation]
+
+    relocate("com.llamalad7.mixinextras", "com.wynntils.mixinextras.shadow")
+    exclude 'META-INF/services/javax.*'
```

Comment:
what javax services do we have? 🤔

Reply:
Forge discord said this is needed, with mixinextras

Reply:
I'll try without I guess.

Reply:
<img width="731" alt="image" src="https://user-images.githubusercontent.com/49001742/212750696-bd1cf84b-1a40-4ff2-aa6b-f70f46a280d3.png">
@magicus I will add a reflecting comment.

Reply:
Okay, it describes why it is bad to have META-INF/services/javax.* files hanging around. I'm still curious: do we have such a file, or is this just a general guidance to other mods that actually have those files? I tried searching but only found the Forge mods.toml in META-INF. But then again, maybe mixinextras produces it itself?

If we don't have it, we don't need to have that exclude. With that said, maybe something will start producing such files in the future, so we can keep the exclude to avoid future surprises, but if so, I think we should add a comment saying that this is strictly speaking not needed right now.

Reply:
I think mixinextras produces this file so it is needed

Reply:
Can you check? If so, a comment about that being the case is also helpful.

Reply:
I could not see it. I updated the comment, but I want to keep it.